src/pages/PreviewPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Header from '../components/header/Header'
+import {userActions} from '../actions/userActions'
+import {connect} from 'react-redux'
+
+const PreviewPage = props => {
+  return (
+    <React.Fragment>
+      <Header />
+    </React.Fragment>
+  )
+}
+
+function mapStateToProps(state) {
+  return {}
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    login: values => dispatch(userActions.login(values))
```

Comment:
Do you need it here in your further development? If so, please, leave some comments. If no, you can remove mapDispatchToProps and mapStateToProps functions.